app/components/layout/UserItem.tsx

```diff
@@ -45,16 +47,55 @@ export function UserItem() {
         </div>
       </Popover>
     );
+  }
+
+  const authError = searchParams.get("authError");
+  const closeAuthErrorDialog = () => {
+    searchParams.delete("authError");
+    setSearchParams(searchParams);
```

Comment:
I believe I missed this first time but I think this should be changed: https://beta.reactjs.org/learn/updating-objects-in-state#treat-state-as-read-only

`searchParams` in this case is state so we should treat it as readonly. Options: first make copy of it before modifying or see if you can give object to setSearchParams as argument or see if you can make functional update work.

Reply:
Yeah, I was being lazy, will update!